src/components/Button/Button.module.css

```diff
@@ -1,14 +1,16 @@
 .Button {
   font-size: var(--button-text);
   padding: 0.625rem 1.6rem;
-  border-radius: var(--default-border-radius);
+  width: 100%;
+  height: 4.375rem;
```

Comment:
Któraś ze zmian w tym pliku (wydaje mi się, że ta) powoduje, że tekst na buttonach nie jest wycentrowany wertykalnie do środka tylko ucieka w dół. Wydaje mi się, że `line-height: 1;` załatwia sprawę.

Reply:
można zrobić tak, albo po prostu usunąć height, bo później jak będzie nam się zmieniał font, to też byśmy pewnie chcieli żeby ten przycisk nie był wtedy taki duży na wysokość a tekst mały. Resztę powinien załatwić padding.